lib/containers/artist.js

```diff
@@ -37,11 +37,19 @@ class Artist extends React.Component {
   render() {
     const windowDimensions = Dimensions.get('window');
     const commonPadding = windowDimensions.width > 700 ? 30 : 20;
+    const tabs = ['ABOUT'];
+    debugger;
+    if (this.props.artist.counts.artworks) {
+      tabs.push('WORKS');
+    }
+    if (this.props.artist.partner_shows.length) {
+      tabs.push('SHOWS');
```

Comment:
from @anipetrov: to my knowledge, no way to associate artists and shows if not through artworks. i am not sure if the artist is still associated with a show if all of his show artworks get unpublished

The update to that logic would be nice with enums in objc but I'm not sure what is more appropriate stylistically with javascript.